pkg/web/pup_router.go

```diff
@@ -13,25 +13,28 @@ import (
 	"github.com/dogeorg/dogeboxd/pkg/conductor"
 )
 
-func NewInternalRouter(config dogeboxd.ServerConfig, dbx dogeboxd.Dogeboxd, pm dogeboxd.PupManager) conductor.Service {
+func NewInternalRouter(config dogeboxd.ServerConfig, dbx dogeboxd.Dogeboxd, pm dogeboxd.PupManager, dkm dogeboxd.DKMManager) conductor.Service {
 	return InternalRouter{
 		config: config,
 		pm:     pm,
 		dbx:    dbx,
 		dbxmux: http.NewServeMux(),
+		dkm:    dkm,
 	}
 }
 
 type InternalRouter struct {
 	config dogeboxd.ServerConfig
 	dbx    dogeboxd.Dogeboxd
 	pm     dogeboxd.PupManager
+	dkm    dogeboxd.DKMManager
 	dbxmux *http.ServeMux
 }
 
 func (t InternalRouter) routes() {
 	t.dbxmux.HandleFunc("POST /dbx/metrics", t.recordMetrics)
 	t.dbxmux.HandleFunc("/dbx/hook/{hookID}", t.hookHandler)
+	t.dbxmux.HandleFunc("POST /dbx/keys/getDelegatedKeys", t.getDelegatedPupKeys)
```

Comment:
Is this dead code? We should probably maybe write the pubkey to `/storage/delegated.key.pub` too